sstables/sstable_directory.hh

```diff
@@ -113,6 +113,8 @@ public:
         virtual future<> process(sstable_directory& directory, process_flags flags) override;
         virtual future<> commit() override;
         virtual future<> prepare(sstable_directory&, process_flags, storage&) override;
+    private:
```

Comment:
This looks like preparation to something, or maybe left over from previous patching. In the former case it deserves separate patch with the indent description, in the latter case the change should be ~~drtopped~~ garbage-collected :)